easybuild/tools/options.py

```diff
@@ -482,6 +482,8 @@ def override_options(self):
                                             None, 'store_true', False),
             'pre-create-installdir': ("Create installation directory before submitting build jobs",
                                       None, 'store_true', True),
+            'prefer-ebpythonprefixes': (("Prefer EBPYTHONPREFIXES over PYTHONPATH when possible"),
```

Comment:
The name of this option is hard to understand, only people with previous knowledge of what `ebpythonprefixes` are will know what this does. I suggest to change it to less technical one.

There are 2 approaches, based on the perspective we pick:
1. Shall Python module have precedence over user installations?
2. Can the user override Python modules installed by EB?

Which could translate to the following options:
1. `--python-module-precedence=[high,low]`
2. `--user-override-python=[true,false]